CODEOWNERS

```diff
@@ -360,7 +365,9 @@ jenkinsfiles @cilium/ci-structure
 /pkg/counter @cilium/sig-datapath
 /pkg/datapath/ @cilium/sig-datapath
 /pkg/datapath/linux/config/ @cilium/loader
+/pkg/datapath/linux/ipsec/ @cilium/ipsec
 /pkg/datapath/linux/ipsec/xfrm_collector* @cilium/metrics
+/pkg/datapath/linux/node.go @cilium/ipsec @cilium/sig-datapath
 /pkg/datapath/linux/probes/ @cilium/loader
```

Comment:
maybe also add `cilium/cmd/encrypt_status.go` to ipsec group?

Reply:
Oh, I thought it was a cmdref and skipped it :facepalm: